app/src/main/java/com/codesoom/assignment/mapper/ProductMapper.java

```diff
@@ -0,0 +1,50 @@
+package com.codesoom.assignment.mapper;
+
+import com.codesoom.assignment.application.dto.ProductCommand;
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.dto.ProductDto;
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.ReportingPolicy;
+import org.mapstruct.factory.Mappers;
+
+@Mapper(
+        componentModel = "spring",
+        injectionStrategy = InjectionStrategy.CONSTRUCTOR,
+        unmappedTargetPolicy = ReportingPolicy.ERROR
+)
+public interface ProductMapper {
```

Comment:
이름으로 팩토리를 써도 괜찮을 것 같습니다.
```suggestion
public interface ProductFactory {
```